editor/effects.js

```diff
@@ -276,4 +288,78 @@ export default {
 
 		return effects;
 	},
+	FETCH_REUSABLE_BLOCKS( action, store ) {
+		const { id } = action;
+		const { dispatch } = store;
+
+		let result;
+		if ( id ) {
+			result = new wp.api.models.ReusableBlocks( { id } ).fetch();
+		} else {
+			result = new wp.api.collections.ReusableBlocks().fetch();
+		}
+
+		result.then(
+			( reusableBlockOrBlocks ) => {
+				dispatch( {
+					type: 'FETCH_REUSABLE_BLOCKS_SUCCESS',
+					reusableBlocks: castArray( reusableBlockOrBlocks ).map( ( { id: itemId, name, content } ) => {
+						const [ { name: type, attributes } ] = parse( content );
+						return { id: itemId, name, type, attributes };
+					} ),
+				} );
+			},
+			( error ) => {
+				dispatch( {
+					type: 'FETCH_REUSABLE_BLOCKS_FAILURE',
+					error: error.responseJSON || {
+						code: 'unknown_error',
+						message: __( 'An unknown error occurred.' ),
+					},
+				} );
+			}
+		);
+	},
+	SAVE_REUSABLE_BLOCK( action, store ) {
+		const { id } = action;
+		const { getState, dispatch } = store;
+
+		const { name, type, attributes } = getReusableBlock( getState(), id );
+		const content = serialize( createBlock( type, attributes ) );
+
+		new wp.api.models.ReusableBlocks( { id, name, content } ).save().then(
```

Comment:
What is the `name` property? The WP REST API [doesn't support](https://developer.wordpress.org/rest-api/reference/posts/#create-a-post) a `name` attribute. Shouldn't it be `title` of the custom post type?

Reply:
`name` is the name of the reusable block: https://github.com/WordPress/gutenberg/blob/master/lib/class-wp-rest-reusable-blocks-controller.php#L302-L307

I agree that naming that `title` is a better thing to call that, though. I'll put it on my list!

Reply:
Ah, I see, thanks.

Your call. :)